### PR TITLE
Various fixes, enhancements

### LINKED ISSUE #21: tonicdb.dbName capitalized in template but not values file
https://github.com/TonicAI/tonic_helm_charts/blob/5f69551566acfe521282f51e5efc2a2a419795b7/templates/tonic-worker-deployment.yaml#L22

When I fixed it tonic paniced because `database "tonic" does not exist`. Slightly confused how it worked in the first place but I fixed it by setting `tonicdb.dbName` to `""`

### PR DESCRIPTION
**Pull request by sblack4**
Various fixes & enhancements to make tonic usable in our env without post-deployment patching. Sorry for such a big PR. 

I fixed #21 in 999500b

Also if you want here's the script I used to retag images

```
#!/bin/bash

NEW_REPO=$1
TONIC_VERION=255
declare -a TONIC_IMAGES=( tonic_notifications tonic_pii_detection tonic_spark_helper tonic_web_server tonic_worker )

for img in ${TONIC_IMAGES[@]}
do
  tonic_full=quay.io/tonicai/$img:$TONIC_VERION
  new_full=$NEW_REPO/$img:$TONIC_VERION
  docker pull $tonic_full
  docker tag $tonic_full $new_full
  docker push $new_full
done
```

### CODE REVIEW

File: values.sample.yaml [hunk 1]
```diff
@@ -25,11 +25,36 @@ enableLogCollection: false
...
+      # This annotation is only required if you are creating an internal facing ELB. Remove this annotation to create public facing ELB.
+      service.beta.kubernetes.io/aws-load-balancer-internal: "true"
+      service.beta.kubernetes.io/azure-load-balancer-internal: "true"
+  worker:
+    image: quay.io/tonicai/tonic_worker
+
 # Use a Tonic license to enable different enterprise features. License keys can be obtained from Tonic.
-# tonicLicense: <license-key>
+# tonicLicenseRaw: <license-key>
+# tonicLicenseSecret: <kubernetes-secret-with-key-tonic-license>
```
Comment (calexander3) [inline values.sample.yaml:57]: The license doesn't contain any sensitive information so I'm not sure if it needs to be a secret. However if we are going to make it one, can you make it always a secret similar to how the database password is done?

File: values.sample.yaml [hunk 2]
```diff
@@ -25,11 +25,36 @@ enableLogCollection: false
 # This value will be provided to you by Tonic and will allow you to authenticate against our private docker repository.
 dockerConfigAuth: <docker-config-auth>
 
+# service account for tonic
+serviceAccount:
+  create: true
```
Comment (calexander3) [inline values.sample.yaml:30]: Should this default to true or false?

File: values.sample.yaml (hunk 1 above)
Comment (sblack4) [reply values.sample.yaml:57]: If you're sure it doesn't need a secret it's simpler to leave it plain text 🤷

File: values.sample.yaml (hunk 2 above)
Comment (sblack4) [reply values.sample.yaml:30]: it can be false, it will just use the default service account for its namespace then.

**Pull request merged**